pkg/encoding/kzgEncoder/multiframe_test.go

```diff
@@ -49,3 +49,45 @@ func TestUniversalVerify(t *testing.T) {
 
 	assert.True(t, group.UniversalVerify(params, samples, numBlob) == nil, "universal batch verification failed\n")
 }
+
+func TestUniversalVerifyWithPowerOf2G2(t *testing.T) {
+	teardownSuite := setupSuite(t)
+	defer teardownSuite(t)
+
+	group, _ := kzgRs.NewKzgEncoderGroup(kzgConfig, true)
+	group.KzgConfig.G2Path = ""
```

Comment:
Isn't the existing test already testing this case, since reading from the Powerof2 file is the default?